bip-emilengler-replacing-the-term-address.mediawiki

```diff
@@ -0,0 +1,46 @@
+<pre>
+  BIP: UNDEFINED
+  Titel: Replacing the term address
+  Author: Emil Engler <me@emilengler.com>
+  Status: Draft
+  Type: Informational
+  Created: 2019-10-17
+  License: CC0
+</pre>
+
+==Abstract==
+This BIP proposes a new term for 'address'
+
+==Specification==
+The new term is:
+''Bitcoin'' '''Invoice''' ''Address''
+
+The term ''Bitcoin'' and ''Address'' are optional this has the following reasons:
+For ''Bitcoin'':
+* Useful for multicoin wallets to indicate that it belongs to Bitcoin
+* Indicates a difference between a lightning and an on-chain invoice
+For ''Address'':
+* To not confuse users with a completely new term
+* To show that it is where you send something to
+
+This gives us the four following possibilities:
+* Bitcoin Invoice Address
+* Bitcoin Invoice
+* Invoice Address
+* Invoice
```

Comment:
"Invoice" alone may be too confused with regular invoices in general.

Reply:
IN the lightning user-land it also just Invoice IIRC